appveyor.yml

```diff
@@ -24,7 +24,9 @@ install:
  - dav1d.cmd
  - libjpeg.cmd
  - zlibpng.cmd
+ - set CC=clang-cl && set CXX=clang-cl
  - libyuv.cmd
+ - set "CC=" && set "CXX="
```

Comment:
Yuan: Do we need to quote `CC=` and `CXX=` in this command?

You did not quote `CC=clang-cl` and `CXX=clang-cl` in line 27.

Reply:
Thanks for the explanation. Then please quote the first one so that the value of `CC` doesn't have a trailing space:

```
- set "CC=clang-cl" && set "CXX=clang-cl"
```

Make the same change to the comment in ext/libyuv.cmd. Thanks.